server/sql.go

```diff
@@ -130,7 +130,7 @@ func (s *Server) registerSQLRoutes(g *echo.Group) {
 				resultSet.Error = err.Error()
 			}
 			// Sync all databases in the instance asynchronously.
-			instanceSyncChan <- instance
```

Comment:
The schema syncer uses a select that processes whenever 1) the channel receives an instance sync or 2) every 30 minutes.

Reply:
But we still gate against maxLastSuccessfulSyncTs in syncAllDatabases.

Reply:
For newly created instances, their databases are created with LastSuccessfulSyncTs setting to 0. Whenever needed, we could even have the frontend to sync the database on-demand when LastSuccessfulSyncTs == 0.

Reply:
Let's say the instance has just synced. Now user clicks the sync instance button, which triggers this API, user would expect us to sync the instance immediately instead of waiting for another almost 30 minutes to sync.

We already have a 30 minutes sync interval, thus I am wondering why we would need another extra LastSuccessfulSyncTs gate.

Reply:
Ah, I see what you mean. I'm expecting the instance sync now button only syncs the instance and the database sync now button to sync the database schema. If we feel the sync instance button to sync all database schema, I can change to that behavior.

Reply:
I feel we just need 2 sync mechanisms.

1. Periodical background sync -  every 30 minutes.
2. Proactive sync - sync whenever code requests to do an immediate sync.

Thus MaxLastSuccessfulSyncTs is not needed

Reply:
Yeah, that makes sense. Updated.